### ISSUE TITLE
Feat: create Subdomain transfer/sale logic

### ISSUE THREAD

**Issue by naftalimurgor** [naftalimurgor/exglos-ens#1]
## Summary

1. Registering 3rd level domains for a `.eth` domains requires that the approval of the owner of the 2LD

For example: if  I have `naftalimurgor.eth` only I the owner can issue 3rd level subdomains `app.naftalimurgor.eth`
This is what we intend to achieve. This can be done by having a backend to approve the issuance of such 3rd level subdomains and a mechanism for achieving the purchase and transfer of the subdomain upon sale.

In summary,
1. claim a 3rd level domain(based on `exglos.eth` for example `foo.exglos.eth` 
2. Checkout with cart option
3. Owner of the exglos.eth (2LD) can then register and approve the domain and transfer this to the new address
